docs/pages/bare/upgrade.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Upgrade your bare project
+description: View file-by-file diffs of all the changes you need to make to your native projects to upgrade them to the next Expo SDK version.
+sidebar_title: Upgrading your bare project
```

Comment:
```suggestion
sidebar_title: Upgrading your bare project
hideTOC: true
```

Reply:
alternatively: what if the sidebar showed files that are included in the diff? might be tricky, can't recall how that's implemented